File: 412 -Fizz Buzz/my.ts
function fizzBuzz(n: number): string[] {
    let response : string[] = []

    for(let i = 1; i <= n; i++) {
        if( i % 5 == 0 && i % 3 == 0) {
            response.push("FizzBuzz")
        } else if( i % 5 == 0) {
            response.push("Buzz")
        } else if( i % 3 == 0) {
            response.push("Fizz")
        } else {
            response.push(`${i}`)
        }
    }
    return response
};

File: 412 -Fizz Buzz/my.py
class Solution:
    def fizzBuzz(self, n: int) -> List[str]:
        response : List[str] = []

        for i in range(1, n +1):
          if i % 5 == 0 and i % 3 == 0:
            response.append("FizzBuzz")
          elif i % 5 == 0:
            response.append("Buzz")
          elif i%3 == 0:
            response.append("Fizz")
          else:
            response.append(str(i))

        return response
        